src/common/kube-helpers.ts

```diff
@@ -182,7 +207,7 @@ export function getNodeWarningConditions(node: V1Node) {
 /**
  * Checks if `config` has valid `Context`, `User`, `Cluster`, and `exec` fields (if present when required)
```

Comment:
Perhaps there should be an annotation about this function returning exceptions instead of throwing them.